docs/yaml_docs/pipeline_visualization_yml.md

```diff
@@ -0,0 +1,161 @@
+full:
+ - long_file1.csv
+ - long_file2.csv
+# the shorter list will be plotted on umaps as well as other plot types, one plot per group
+minimal:
+ - short_file1.csv
+
+```
+<span class="parameter">custom_markers</span><br>
+  - <span class="parameter">files</span><br>
+  
+    - <span class="parameter">full:</span><br>
+The full list will be plotted in dot plots and matrix plots, with one plot per group. 
+
+     - <span class="parameter">minimal:</span><br>
+The shorter list will be plotted on umaps as well as other plot types, with one plot per group. 
+
+
+- <span class="parameter">paired_scatter:</span>`String`, Default:  <br>
+  Where different normalization exists in a modality, choose which one to use, set X or leave blank to use the mdata[mod].X assay. 
+
+- <span class="parameter">layers:</span><br>
+  - <span class="parameter">rna:</span>`String`, Default: logged_counts<br>
+  - <span class="parameter">prot:</span>`String`, Default: clr<br> 
+  - <span class="parameter">atac:</span>`String`, Default: signac_norm<br> 
+Check [gene_list_format.md](https://github.com/DendrouLab/panpipes/edit/clustering_g/docs/usage/gene_list_format.md) for Plot marker csv format instructions. 
+
+
+## Plot metadata variables 
+
+- <span class="parameter">categorical_vars:</span>`String`, Default: &categorical_vars<br>
+  - <span class="parameter">all:</span>`String`, Default: rep:receptor_subtype  sample_id<br>
+Metrics to be plotted on every modality. 
+  - <span class="parameter">rna:</span>`String`, Default: rna:predicted_doublets  rna:phase<br>
+  - <span class="parameter">prot:</span>`String`, Default: prot:leiden_res0.2    prot:leiden_res1<br>
+  - <span class="parameter">atac:</span>`String`, Default: <br>
+  - <span class="parameter">rep:</span>`String`, Default: rep:has_ir<br>
+  - <span class="parameter">multimodal:</span>`String`, Default: leiden_totalVI    mdata_colsr<br> 
+
+- <span class="parameter">continuous_vars:</span>`String`, Default: &continuous_vars<br>
+  - <span class="parameter">all:</span>`String`, Default:leiden_res0.5<br>
+Metrics to be plotted on every modality. 
+  - <span class="parameter">rna:</span>`String`, Default: rna:total_counts<br>
+  - <span class="parameter">prot:</span>`String`, Default: prot:total_counts<br>
+  - <span class="parameter">atac:</span>`String`, Default: <br>
+  - <span class="parameter">multimodal:</span>`String`, Default: rna:total_counts    prot:total_counts<br>
+  
+- <span class="parameter"paired_scatter:</span>`String`, Default: scatter_features.csv<br>
+Check [gene_list_format.md](https://github.com/DendrouLab/panpipes/edit/clustering_g/docs/usage/gene_list_format.md) for metadata csv format instructions. 
+
+## Plot style 
+Choose the plot type desired. 
+- <span class="parameter">do_plots:</span><br>
+  - <span class="parameter">categorical_barplots:</span>`Boolean`, Default: True<br>
+  Plot each categorical variable as a bar plot. 
```

Comment:
for each of thse can you the type of plot being plotting above the param, rather than underneath it?